extensions/elytron-security-oauth2/runtime/src/main/java/io/quarkus/elytron/security/oauth2/runtime/oauth/OAuth2IdentityManager.java

```diff
@@ -0,0 +1,83 @@
+package io.quarkus.elytron.security.oauth2.runtime.oauth;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.jboss.logging.Logger;
+import org.wildfly.security.auth.server.RealmUnavailableException;
+import org.wildfly.security.auth.server.SecurityDomain;
+import org.wildfly.security.auth.server.SecurityIdentity;
+import org.wildfly.security.authz.RoleDecoder;
+import org.wildfly.security.authz.Roles;
+import org.wildfly.security.evidence.BearerTokenEvidence;
+
+import io.undertow.security.idm.Account;
+import io.undertow.security.idm.Credential;
+import io.undertow.security.idm.IdentityManager;
+
+public class OAuth2IdentityManager implements IdentityManager {
+    private static Logger log = Logger.getLogger(OAuth2IdentityManager.class);
+    private final SecurityDomain domain;
+    private String roleClaim;
+    private String roleSeparator;
+
+    public OAuth2IdentityManager(SecurityDomain domain, String roleClaim, String roleSeparator) {
+        this.domain = domain;
+        this.roleClaim = roleClaim;
+        // configs are trimmed so we use ' ' instead of a space string as default value
+        this.roleSeparator = roleSeparator == "' '" ? " " : roleSeparator;
+        System.out.println(roleClaim + roleSeparator);
```

Comment:
Debug log it please instead

Reply:
Sorry, I push my commit too quickly it was a _developement debug line_. I removed it